pkg/query-service/model/v3/v3.go

```diff
@@ -238,6 +237,7 @@ type AttributeKey struct {
 	Key      string               `json:"key"`
 	DataType AttributeKeyDataType `json:"dataType"`
 	Type     AttributeKeyType     `json:"type"`
+	IsColumn bool                 `json:"isColumn"`
```

Comment:
Added `IsColumn` param too as it was discussed previously.

Reply:
I forgot what was the reason to remove it from `AttributeKeyType` to have a separate field?

Reply:
In future we are going to add few resource attributes as permanent columns and in that case for those resource attributes `AttributeKeyType` will be both `resource` and `column` so we agreed to add `IsColumn` to avoid confusion.